.github/workflows/_linux_ut.yml

```diff
@@ -120,7 +122,7 @@ jobs:
             if [[ "$xpu_case" != *"*"* && "$xpu_case" != *.so && "$xpu_case" != *.a ]]; then
               case_name=$(basename "$xpu_case")
               echo "Testing ${case_name} ..."
-              "$xpu_case" --gtest_output=xml:"$TEST_REPORTS_DIR"/"$case_name".xml 
+              "$xpu_case" --gtest_output=xml:"$TEST_REPORTS_DIR"/"$case_name".xml 2>&1 | tee ${{ github.workspace }}/ut_log/binary_UT_${ut_suite}_${case_name}_Test.log
```

Comment:
add one more folder under `ut_log`, for example `ut_log/binary_ut/${case_name}...log`